From a749fbfd4a392511476b2bb2df9c9dd93d647413 Mon Sep 17 00:00:00 2001
From: Jon Phillips <jphillips@cloudflare.com>
Date: Thu, 9 May 2024 23:36:43 +0100
Subject: Rename V8_COMPRESS_POINTERS_IN_ISOLATE_CAGE ->
 V8_COMPRESS_POINTERS_IN_MULTIPLE_CAGES in BUILD.bazel.

This appears to have been missed as part of the v8 patch which applied
this rename elsewhere:
https://chromium-review.googlesource.com/c/v8/v8/+/5279595.

diff --git a/BUILD.bazel b/BUILD.bazel
index e99cb6fe2dc950678612d22693fd8dbfa8a3aa7a..4146cef20a6684ae811aacc7a93a0832943a2173 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -329,10 +329,10 @@ selects.config_setting_group(
     ],
 )
 
-# Enable isolated cage if v8_enable_pointer_compression and
+# Enable multiple cages if v8_enable_pointer_compression and
 # NOT v8_enable_pointer_compression_shared_cage.
 selects.config_setting_group(
-    name = "enable_pointer_compression_isolated_cage",
+    name = "enable_pointer_compression_multiple_cages",
     match_all = [
         ":is_v8_enable_pointer_compression",
         ":is_not_v8_enable_pointer_compression_shared_cage",
@@ -529,8 +529,8 @@ v8_config(
         ":enable_pointer_compression_shared_cage": [
             "V8_COMPRESS_POINTERS_IN_SHARED_CAGE",
         ],
-        ":enable_pointer_compression_isolated_cage": [
-            "V8_COMPRESS_POINTERS_IN_ISOLATE_CAGE",
+        ":enable_pointer_compression_multiple_cages": [
+            "V8_COMPRESS_POINTERS_IN_MULTIPLE_CAGES",
         ],
         "//conditions:default": [],
     }) + select({
